File: week11_test/revison/devowel_file.py
#!/usr/bin/python3

import sys, re

for file in sys.argv[1:]:
	with open(file) as f:
		content = f.read()
	content = re.sub('[aeiuo]','',content,flags=re.I)
	with open(file,"w") as f:
		f.write(content)

File: week11_test/revison/devowel_file.pl
#!/usr/bin/perl -w

use strict;

for my $arg (@ARGV){
	open F, '<', $arg or die "Cannot open $arg \n";
	my @a = <F>;
	close F;
	foreach my $line (@a){
		$line =~ s/[aeiou]//gi;
	}
	open OUT, '>', $arg or die "Write to $arg failed!\n";
	print OUT @a;
	close OUT;
	
}
